core/src/epicli/.devcontainer/Dockerfile

```diff
@@ -65,3 +65,5 @@ RUN : SETUP USER, CERTS AND OTHERS \
  && chmod ug=r,o= /etc/sudoers.d/$USERNAME \
  && setcap 'cap_net_bind_service=+ep' /usr/bin/ssh \
  && /bin/bash /config-post.sh
+
+RUN echo alias epicli='"cd /workspaces/epiphany/core/src/epicli && PYTHONPATH=$(pwd) python3 cli/epicli.py"' >> /etc/bash.bashrc
```

Comment:
Its fine, but this could be even better:
alias epicli='PYTHONPATH=$PYTHONPATH:/workspaces/epiphany/core/src/epicli/; python3 /workspaces/epiphany/core/src/epicli/cli/epicli.py'
1- do not override global env pythonpath,
2- do not change user pwd (can be annoying)

Reply:
As you could see after some testing, pwd needs to be /workspaces/epiphany/core/src/epicli. Also the PYTHONPATH is only for that session so should not matter.

Reply:
alias epicli="export PYTHONPATH=/workspaces/epiphany/core/src/epicli && python3 -m cli.epicli"
should do the job, can you verify?

Reply:
Or alternative approach:
```
cat /etc/profile.d/epicli.sh
export PYTHONPATH=${PYTHONPATH}:/workspaces/epiphany/core/src/epicli
alias epicli='python3 -m cli.epicli'
```

Reply:
That fixed the issue. For now lets leave as is.